pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -133,6 +137,8 @@ type mergeQuerier struct {
 // For the label "original_" + `idLabelName it will return all the values
 // of the underlying queriers for `idLabelName`.
 func (m *mergeQuerier) LabelValues(name string, matchers ...*labels.Matcher) ([]string, storage.Warnings, error) {
+	log, _ := spanlogger.New(m.ctx, "mergeQuerier.labelValues")
```

Comment:
to avoid confusion I'd keep the case of the method name (`L` instead of `l`)
```suggestion
	log, _ := spanlogger.New(m.ctx, "mergeQuerier.LabelValues")
```

Same for the other two below

Reply:
done